app/src/main/java/com/fmpdroid/moneybox/MainActivity.java

```diff
@@ -0,0 +1,57 @@
+package com.fmpdroid.moneybox;
+
+import androidx.appcompat.app.ActionBarDrawerToggle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.core.view.GravityCompat;
+import androidx.drawerlayout.widget.DrawerLayout;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
+
+    private DrawerLayout drawer;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        if (getIntent().getBooleanExtra(this.getResources().getString(R.string.key_isEmpty), true)) {
+            Intent myIntent = new Intent(this, EmptyActivity.class);
+            MainActivity.this.startActivity(myIntent);
+        } else {
+            goToDrawerScreen();
+        }
+
+    }
+
+    private void goToDrawerScreen() {
+        Toolbar toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayShowTitleEnabled(true);
+        getSupportActionBar().setTitle("Money Box");
```

Comment:
Put "Money Box" in an enum. Moving forward, activity titles should be stored in this enum.
Lastly, all setTitle's value should be stored in this newly created enum.